src/controller/AutoCommissioner.h

```diff
@@ -133,6 +137,9 @@ class AutoCommissioner : public CommissioningDelegate
     uint8_t mAttestationElements[Credentials::kMaxRspLen];
     uint16_t mAttestationSignatureLen = 0;
     uint8_t mAttestationSignature[Crypto::kMax_ECDSA_Signature_Length];
+
+    uint8_t mICDSymmetricKey[16];
+    uint8_t mICDVerificationKey[16];
```

Comment:
We shouldn't use Crypto constants for the length instead of hardcode values.

Reply:
Done